e2e/utils.go

```diff
@@ -1580,10 +1580,10 @@ func k8sVersionGreaterEquals(c kubernetes.Interface, major, minor int) bool {
 		// return value.
 	}
 
-	maj := strconv.Itoa(major)
-	min := strconv.Itoa(minor)
+	_maj := strconv.Itoa(major)
```

Comment:
This is rather ugly, not only because of the added `_`, also because comparing versions as a string.

Maybe it makes more sense to convert `v.Major` and `v.Minor` to integers?